### DISCUSSION TITLE
Select 2nd Header value with identical name in Tag

### DISCUSSION THREAD

**Discussion by jmevel** [Kong/insomnia#6363] [Help]
Hello,

I'm quite new to Insomnia and I love it so far.
However today I'm stuck on something.

I have this `Response` from the server
![Response](https://github.com/Kong/insomnia/assets/2572775/3a1df646-04e0-4ee8-9441-69220a5d5c71)

As you can see there're two `set-cookie` headers. I don't care about the first one, I only want to be able to extract the second one in a variable inside my `Base environment` so I can chain calls always using the latest `XSRF-TOKEN`.

But I can't figure out how to reference this second `set-cookie` header when I edit my tag, it's by default referencing the first one
![Tag](https://github.com/Kong/insomnia/assets/2572775/9b731282-ec10-4838-adfe-5ef5c1d6ecf1)

I tried `set-cookie[1]` but this is invalid and I get this error message in the `Live Preview` area

>No header with name "set-cookie[1]".
Choices are [
	"content-type",
	"date",
	"cache-control",
	"pragma",
	"set-cookie",
	"set-cookie",
	"x-datadome",
	"accept-ch",
	"strict-transport-security",
	"referrer-policy",
	"x-xss-protection",
	"x-content-type-options",
	"content-security-policy"
]

Even better instead of selecting a header based on its index, it would be awesome if I could select it with a Regex or something similar so I could always select the one starting with `XSRF-TOKEN=` for example.

Is there a way to do this already?
If not, would that be possible to implement this new feature?

Thanks for your answers
(2023.5.6 is a great update by the way, just got it this morning)

**Comment by OxnardM** [comment]
I guess u got a solution? seems like it is 2023.5.6 but unsure how to find that. How do I find 2023.5.6?

**Reply by jmevel** [reply]
I have no idea if this is fixed on newer versions of Insomnia. I don't really use it anymore, my company doesn't allow us to upgrade so I still have version `2023.5.8` installed. 

But judging by the number of votes on this thread and the lack of answers I'd guess this issue is still present in the latest version.
